### PR TITLE
[Doc] Show how to create repository in Aliyun OSS

### PR DESCRIPTION
**Pull request by sduzh**
Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5
  - [ ] 2.4

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
#### [BE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

**Comment by wanpengfei-git** [comment]
#### [FE Incremental Coverage Report]


:heart_eyes: **pass** : 0 / 0 (0%)

File: docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md [hunk 1]
```diff
@@ -62,6 +62,13 @@ StarRocks supports creating repositories in HDFS, AWS S3, and Google GCS.
 >
 > StarRocks supports creating repositories in Google GCS only according to the S3A protocol. Therefore, when you create repositories in Google GCS, you must replace the prefix in the GCS URI you pass as a repository location in `ON LOCATION` with `s3a://`.
 
+
+- For Aliyun OSS:
+  - "fs.oss.accessKeyId": The Access Key ID that you can use to access the Aliyun OSS bucket. 
+  - "fs.oss.accessKeySecret": The Secret Access Key that you can use to access the Aliyun OSS bucket.
+  - "fs.oss.endpoint": The endpoint that you can use to access the Aliyun OSS bucket.
```
Comment (DanRoscigno) [inline docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md:69]:
```suggestion
  - "fs.oss.endpoint": The endpoint used to access the Aliyun OSS bucket.
```

A little shorter

File: docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md [hunk 2]
```diff
@@ -62,6 +62,13 @@ StarRocks supports creating repositories in HDFS, AWS S3, and Google GCS.
 >
 > StarRocks supports creating repositories in Google GCS only according to the S3A protocol. Therefore, when you create repositories in Google GCS, you must replace the prefix in the GCS URI you pass as a repository location in `ON LOCATION` with `s3a://`.
 
+
+- For Aliyun OSS:
+  - "fs.oss.accessKeyId": The Access Key ID that you can use to access the Aliyun OSS bucket. 
+  - "fs.oss.accessKeySecret": The Secret Access Key that you can use to access the Aliyun OSS bucket.
```
Comment (DanRoscigno) [inline docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md:68]:
```suggestion
  - "fs.oss.accessKeySecret": The Secret Access Key used to access the Aliyun OSS bucket.
```

File: docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md [hunk 3]
```diff
@@ -62,6 +62,13 @@ StarRocks supports creating repositories in HDFS, AWS S3, and Google GCS.
 >
 > StarRocks supports creating repositories in Google GCS only according to the S3A protocol. Therefore, when you create repositories in Google GCS, you must replace the prefix in the GCS URI you pass as a repository location in `ON LOCATION` with `s3a://`.
 
+
+- For Aliyun OSS:
+  - "fs.oss.accessKeyId": The Access Key ID that you can use to access the Aliyun OSS bucket.
```
Comment (DanRoscigno) [inline docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md:67]:
```suggestion
  - "fs.oss.accessKeyId": The Access Key ID used to access the Aliyun OSS bucket. 
```

File: docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md (hunk 1 above)
Comment (sduzh) [reply docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md]: Thank you for your suggestion. I have updated the wording here based on your feedback, and also made similar adjustments in other places with the same issue.

File: docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md [hunk 4]
```diff
@@ -54,14 +54,21 @@ StarRocks supports creating repositories in HDFS, AWS S3, and Google GCS.
...
+  - "fs.s3a.access.key": The Access Key used to access the Google GCS bucket.
+  - "fs.s3a.secret.key": The Secret Key used to access the Google GCS bucket.
+  - "fs.s3a.endpoint": The endpoint used to access the Google GCS bucket.
 
 > **NOTE**
 >
 > StarRocks supports creating repositories in Google GCS only according to the S3A protocol. Therefore, when you create repositories in Google GCS, you must replace the prefix in the GCS URI you pass as a repository location in `ON LOCATION` with `s3a://`.
 
+
+- For Aliyun OSS:
```
Comment (kevincai) [inline docs/sql-reference/sql-statements/data-definition/CREATE REPOSITORY.md:66]: there are two ways to access oss, 1) fs.oss.xxx which uses the jindo sdk 2) aws.s3.xxx, which uses aws s3 sdk. we may need to mention both? if jindosdk is not working in some cases.

**Review by kevincai — APPROVED** [review]

**Comment by sduzh** [comment]
I am going to close this PR because I was informed that the lack of documentation about Aliyun OSS in the English docs was an intentional omission.

**Pull request closed without merging**